test/test_table_filtering.py

```diff
@@ -0,0 +1,226 @@
+from aiven_db_migrate.migrate.pgmigrate import PGMigrate
+from test.conftest import PGRunner
+from test.utils import random_string, Timer
+from typing import Dict, Set, Tuple
+
+import psycopg2
+import pytest
+
+
+@pytest.mark.parametrize(["skip", "with_extension"], [
+    [False, False],
+    [True, True],
+    [True, False],
+    [False, True],
```

Comment:
Nitpick: this also could be done like `itertools.product([True, False], repeat=2)`

Reply:
will leave it the way it is for now (visibility)